src/entity.ts

```diff
@@ -222,6 +222,17 @@ export namespace entity {
    *   namespace: 'My-NS',
    *   path: ['Company', 123]
    * });
+   *
+   * @example
+   * <caption>Serialize the key for later re-use.</caption>
```

Comment:
I'm not sure if this would impact the formatting, but usually caption is listed on the same line as the example tag. Might want to generate this locally and check